Containers/mastercontainer/start.sh

```diff
@@ -20,6 +20,11 @@ case "${1}" in
 esac
 }
 
+if [ "$(whoami)" != "root" ] || [ "$(id -u)" != "0" ] || [ "$(id -g)" != "0" ]; then
+    print_red "Container does not run as root. Cannot continue."
```

Comment:
```suggestion
# Check if running as root user
if [ "$EUID" != "0" ]; then
    print_red "Container does not run as root user. This is not supported."
```

Reply:
I've added this changed and I didn't see the change of the if condition, is EUID even a valid env?

Reply:
Yes, it is. Just try it :)

Reply:
what is there the difference to id -u / id -g / whoami?

Reply:
Both would work but using the variable seems cleaner to me :)

Reply:
but what if the group is different?

Reply:
I think that doesnt matter since you have root permissions anyway?

Reply:
not sure...